doc/help.rst

```diff
@@ -2,11 +2,56 @@
 Help
 ####
 
-You can use the **Help** pane to find, render and display rich documentation for any object with a docstring, including modules, classes, functions and methods.
-Help can be retrieved both by static analysis of open files in the :doc:`editor`, or by dynamically inspecting an object in an :doc:`ipythonconsole`.
+You can use the **Help** pane to find, render and display rich documentation for any object with a docstring, including modules, classes, functions and methods. This allows you to access documentation easily directly from Spyder not having to break your workflow.
```

Comment:
```suggestion
You can use the **Help** pane to find, render and display rich documentation for any object with a docstring, including modules, classes, functions and methods.
This allows you to access documentation easily directly from Spyder, without having to interrupt your workflow.
```

Reply:
Also fixed some grammar issues with the last sentence (I fixed them originally but somehow the suggestion got rejected, lost or overwritten).